src/SeedCppCore/Services/System/ResourceService.cpp

```diff
@@ -9,23 +9,41 @@
 
 
 namespace seed_cpp { namespace service {
+	
+	const std::string ResourceService::schemaPath			= "JSONSchemas/";
 
-	const std::string ResourceService::schemaPath = "JSONSchemas/";
-	const std::string ResourceService::schemaPatientPost = "Endpoint/PatientsPostEndpoint.json";
-	const std::string ResourceService::schemaPatientPut = "Endpoint/PatientsPutEndpoint.json";
-	const std::string ResourceService::schemaUserPost = "Endpoint/UsersPostEndpoint.json";
-	const std::string ResourceService::schemaUserPut = "Endpoint/UsersPutEndpoint.json";
-	const std::string ResourceService::schemaModelPatient = "Model/Patient.json";
-	const std::string ResourceService::schemaModelAddress = "Model/Address.json";
-	const std::string ResourceService::schemaModelUser = "Model/User.json";
+	// Endpoint schemas
+	const std::string ResourceService::schemaPatientPost	= "Endpoint/PatientsPostEndpoint.json";
+	const std::string ResourceService::schemaPatientPut		= "Endpoint/PatientsPutEndpoint.json";
+	const std::string ResourceService::schemaUserPost		= "Endpoint/UsersPostEndpoint.json";
+	const std::string ResourceService::schemaUserPut		= "Endpoint/UsersPutEndpoint.json";
+	const std::string ResourceService::schemaAllergyPost	= "Endpoint/AllergiesPostEndpoint.json";
+	const std::string ResourceService::schemaAllergyPut		= "Endpoint/UsersPutEndpoint.json";
```

Comment:
Path to the allergies PUT endpoint is not correct

Reply:
Copy-paste error I missed. Fixed now.